repository/RediStick-Connection-Pool.package/RsRedisConnectionPool.class/instance/createStickAt..st

```diff
@@ -6,4 +6,5 @@ createStickAt: key
 	stick := RsRediStick targetUrl: self primaryUrl.
 	stick beSticky.
 	stick connect.
+	self authentificateStick: stick.
```

Comment:
I think `stick authenticate` would be better.
Stick itself should have authentication feature. Connection pool's responsibility is keeping connections.

Reply:
commit 940d78a